libs/components/tabs/src/lib/modules/vertical-tabset/vertical-tabset-group.component.ts

```diff
@@ -98,14 +122,20 @@ export class SkyVerticalTabsetGroupComponent implements OnInit, OnDestroy {
   };
 
   public tabsHidden = () => {
-    // this fixes an animation bug with ngIf when the parent component goes from visible to hidden
-    this.#openBeforeTabsHidden = this.open;
-    this.open = false;
+    // Angular will sometimes place the animation into the "void" state when tabs are hidden. Update our internal variable to reflect that.
+    this.slideDirection = 'void';
     this.#changeRef.markForCheck();
   };
 
   public tabsShown = () => {
-    this.open = this.#openBeforeTabsHidden;
+    // Set the animation back up so that the "void" state is returned to where it was prior to the tabs being hidden.
+    // This will be instantaneous due to there not being a "void -> *" state on the slide animation.
+    this.#slideForExpanded(true);
     this.#changeRef.markForCheck();
   };
+
+  #slideForExpanded(animate: boolean): void {
```

Comment:
the naming of this is confusing me a little. Is it specifically setting animation related properties only when the tabs are expanded? Is it triggering the animation?